app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h2> This is a test </h2>
\ No newline at end of file
+<section class='posts'>
+  <%= render'partials/user', user: @user %>
+    <% @posts.each do |post| %>
+   <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+    <ul class="post-list">
+    <%= render 'partials/post', post: post %></ul>
```

Comment:
Wireframe shows that `comments counter` and `likes counter` are on right side in every post and comments text is on left. KIndly compare your page layout with wireframe and design the page as it's shown.


![compare](https://user-images.githubusercontent.com/35504149/205401494-69e63ce9-46d5-4326-b49f-9828be805311.png)